bot/plotting/comparison_generator.py

```diff
@@ -6,6 +6,58 @@
 from experiment.experiment_generator import experiment_generator
 
 
+param_to_vary_dict = {
```

Comment:
Here, is this implementation good? or should I do something like this -
```py
param_to_vary_dict = {
    "Current function [A]": {},
    "Electrode height [m]": {},
    "Electrode width [m]": {
        "lower_bound": 0.1,
        "upper_bound": None
    }
}
```

Reply:
Either is good. I'm not sure if there is a recommended style for this?

Reply:
I guess you could do a dict like `{"name": (lower_bound, upper_bound)}`, which would be a little more compact.